code/game/machinery/_machines_base/stock_parts/network_lock.dm

```diff
@@ -0,0 +1,135 @@
+/obj/item/stock_parts/network_lock
+	name = "network access lock"
+	desc = "An id-based access lock preventing tampering with a machine's hardware  and software. Connects wirelessly to network."
```

Comment:
`hardware  and software` odd space

Reply:
makes sense from a gameplay perspective. no need to split them out, obviously wouldn't work in real life.

Reply:
Oh you meant LITERALLY an odd space. I didn't even see it.

Reply:
I mean, check the code, there's to spaces between `hardware` and `and`